cca_realm_claims.go

```diff
@@ -0,0 +1,242 @@
+// CCA Realm Claims
+package ccatoken
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/veraison/eat"
+)
+
+type CcaRealmClaims struct {
+	Challenge              *eat.Nonce `cbor:"10,keyasint" json:"cca-realm-challenge"`
+	PersonalizationValue   *[]byte    `cbor:"44235,keyasint" json:"cca-realm-personalization-value"`
+	InitialMeasurements    *[]byte    `cbor:"44238,keyasint" json:"cca-realm-initial-measurement"`
+	ExtensibleMeasurements *[][]byte  `cbor:"44239,keyasint" json:"cca-realm-extensible-measurements"`
+	HashAlgID              *string    `cbor:"44236,keyasint" json:"cca-realm-hash-algo-id"`
+	PubicKey               *[]byte    `cbor:"44237,keyasint" json:"cca-realm-public-key"`
```

Comment:
Typo on this struct member

Reply:
@SimonFrost-Arm Thanks, I will check this,,,